geode-assembly/build.gradle

```diff
@@ -113,6 +113,14 @@ publishing {
   }
 }
 
+gradle.taskGraph.whenReady( { graph ->
+  tasks.withType(Zip).each { zip ->
+    if ( "task ':geode-assembly:dockerfileZip'" == "${zip}") {
+      zip.enabled = false
+    }
+  }
+})
+
```

Comment:
Instead, we can add the task to the list of other zip tasks we are disabling:
```
[tasks.named('distZip'),
 tasks.named('srcDistZip'),
 tasks.named('dockerfileZip')
 ]*.configure {
  enabled = false
}
```

Reply:
nice, that is much cleaner!